api/datasources.production.js

```diff
@@ -1,12 +0,0 @@
-module.exports = {
```

Comment:
See datasources.local.js, can we keep the file but move secure values to `process.env` through the `.env` file? Less moving parts when getting the stack spun up.